skyvern/webeye/scraper/domUtils.js

```diff
@@ -563,6 +563,16 @@ function getListboxOptions(element) {
   return selectOptions;
 }
 
+function uniqueId() {
+    const characters = 'ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789';
```

Comment:
I wonder if making the unique IDs case insensitive would prevent LLMs from hallucinating.

Reply:
I tested with Claude 3 Sonnet and GPT4O on my local, didn't hallucinate so far. Let's keep it this way but we should keep an eye out for it @Skyvern-AI/eng